_docs_integrate/error-codes.md

```diff
@@ -24,74 +24,163 @@ required_by:
+| {% include anchor a="error.consumption.attributes.successorSourceContentIsNotEqualToCopyContent" %}             | Successor source attribute content doesn't match successor shared attribute copy. ([Update Attributes by succession]({% link _docs_integrate/update-attributes-by-succession.md %}))                                                                                                                     |
+
+## ErrorCodes with the prefix "error.consumption.requests." and the ErrorCode "inheritedFromItem"
+
+| ErrorCode                                                                                                   | Description                                                                                                                                                                                                                                                                                                                                           |
+| ----------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| {% include anchor a="inheritedFromItem" %}                                                                  | Some child items have errors. For more details you have to validate your request. (([Check if outgoing request can be created]({% link _docs_use-cases/use-case-consumption-check-if-outgoing-request-can-be-created.md %}), [Test your requests validity]({% link _docs_integrate/integration-example.md %}#connector-test-your-requests-validity))) |
+| {% include anchor a="error.consumption.requests.decide.validation.invalidNumberOfItems" %}                  | The Request and the Response item count is different. ([Request and response introduction]({% link _docs_integrate/request-and-response-introduction.md %}))                                                                                                                                                                                          |
+| {% include anchor a="error.consumption.requests.decide.validation.itemAcceptedButParentNotAccepted" %}      | You accepted an item of a Request or RequestItemGroup, but you rejected the parent. If you accept a RequestItem, you need to accept all parents up to the root. ([Working with required and optional requestitems]({% link _docs_integrate/request-and-response-introduction.md %}#working-with-required-and-optional-requestitems))                  |
```

Comment:
I would leave it like it is because @britsta has changed this errors for V5 and when we will merge V5 we can do another look over all error codes changes.